overlay/ports/ffmpeg/portfile.cmake

```
@@ -0,0 +1,1081 @@
+        0003-fix-windowsinclude.patch
+        0004-dependencies.patch
+        0005-fix-nasm.patch
+        0007-fix-lib-naming.patch
+        0013-define-WINVER.patch
+        0020-fix-aarch64-libswscale.patch
+        0024-fix-osx-host-c11.patch
+        0040-ffmpeg-add-av_stream_get_first_dts-for-chromium.patch # Do not remove this patch. It is required by chromium
+        0041-add-const-for-opengl-definition.patch
+        0043-fix-miss-head.patch
+        # FIXME should be only applied if feature "fedora-ffmpeg-free-safe" is used
+        0099-ffmpeg-allow-fdk-aac-free.patch
+)
+
+set(OPTIONS "--enable-pic --disable-doc --enable-debug --enable-runtime-cpudetect --disable-autodetect")
+
+if("fedora-ffmpeg-free-safe" IN_LIST FEATURES)
+    set(FEDORA_FFMPEG_FREE_SAFE ON)
+    file(DOWNLOAD "https://src.fedoraproject.org/rpms/ffmpeg/raw/f41/f/ffmpeg_free_sources" "${CURRENT_BUILDTREES_DIR}/ffmpeg_free_sources")
+    file(DOWNLOAD "https://src.fedoraproject.org/rpms/ffmpeg/raw/f41/f/enable_decoders" "${CURRENT_BUILDTREES_DIR}/enable_decoders")
+    file(DOWNLOAD "https://src.fedoraproject.org/rpms/ffmpeg/raw/f41/f/enable_encoders" "${CURRENT_BUILDTREES_DIR}/enable_encoders")
+
+    file(STRINGS "${CURRENT_BUILDTREES_DIR}/ffmpeg_free_sources" FFMPEG_SOURCE_ALLOW_LIST)
+    file(STRINGS "${CURRENT_BUILDTREES_DIR}/enable_decoders" FFMPEG_ENABLE_DECODERS_DIRTY)
+    file(STRINGS "${CURRENT_BUILDTREES_DIR}/enable_encoders" FFMPEG_ENABLE_ENCODERS_DIRTY)
+
+    set(FFMPEG_ENABLE_DECODERS "")
+    set(FFMPEG_ENABLE_ENCODERS "")
+
+    foreach(decoder_dirty IN LISTS FFMPEG_ENABLE_DECODERS_DIRTY)
+        if(decoder_dirty MATCHES "^([^#]+)#?.*$")
+            string(STRIP "${CMAKE_MATCH_1}" decoder)
+            list(APPEND FFMPEG_ENABLE_DECODERS "${decoder}")
+        endif()
+    endforeach()
+    foreach(encoder_dirty IN LISTS FFMPEG_ENABLE_ENCODERS_DIRTY)
+        if(encoder_dirty MATCHES "^([^#]+)#?.*$")
+            string(STRIP "${CMAKE_MATCH_1}" encoder)
+            list(APPEND FFMPEG_ENABLE_ENCODERS "${encoder}")
+        endif()
+    endforeach()
+
+    # Manually whitelisting some low risk sources
+    list(APPEND FFMPEG_SOURCE_ALLOW_LIST 
+        "libavcodec/file_open.c"
+        "libavdevice/file_open.c"
+        "libavfilter/file_open.c"
+        "libavformat/file_open.c"
+    )
+
+    # Filter out source files directly from filesystem to ensure they aren't accidentally included
+    file(GLOB_RECURSE FFMPEG_SOURCE_FILE RELATIVE "${SOURCE_PATH}" "${SOURCE_PATH}/*")
+    message(STATUS "Filter source that is not part of Fedora's ffmpeg-free")
+    foreach(srcfile IN LISTS FFMPEG_SOURCE_FILE)
+        if (NOT(srcfile IN_LIST FFMPEG_SOURCE_ALLOW_LIST) AND NOT(srcfile MATCHES "compat/.*"))
+            message(STATUS "-- ${srcfile} is NOT in the allowlist. Removing")
+            file(REMOVE "${SOURCE_PATH}/${srcfile}")
+        endif()
+    endforeach()
+
+    # Encoder/decoder for which the source has been blacklisted
```

```suggestion
    # Encoder/decoder for which the source has been removed
```
Is this correct? At least some ac3 files are listed as free: https://src.fedoraproject.org/rpms/ffmpeg/blob/f41/f/ffmpeg_free_sources#_296